### ISSUE TITLE
Design the article page for mobile + tablet

### ISSUE THREAD

**Issue by gissoo** [Princeton-CDH/startwords#40]
must contain:
- all article metadata 
- download/view to alternate versions (links)
- main body 
- images + icons (at least one image for sampling)
- link style
- quotation style
- contextual note style
- nav

**Comment by gissoo** [comment]
@gwijthoff Here is the designs for the article page for mobile and tablet
(https://www.figma.com/file/MEt3gfKX4N3WTLPbLGBbGC/Startwords?node-id=153%3A2295)
Please let me know what you think!!


Note: I am yet to work on the footer (the license icons)

**Comment by thatbudakguy** [comment]
two comments:
- the date format seems a little odd. I don't see space for a month. maybe we could settle on something standard like "Jan. 5, 2014" or just "2014-01-05".
- i feel like we could relax the margins on the article metadata at the top a little bit on tablet, especially since titles are likely to get pretty long sometimes (and there may be many authors, like here).

**Comment by gwijthoff** [comment]
The date should be consistent throughout the site, the standard for academic journals is either season (Spring/Summer etc) or month only. So, the dates should always be April 2020, for example, throughout the site.

**Comment by gissoo** [comment]
@thatbudakguy The format of the date I have used is "Mon. 00, 0000" – in case that is not clear I meant this "Apr. 11, 2020" 

@gwijthoff So I agree the issue could say "April 2020", however do you think the article itself should also only mention the month and skip the day of the month?

**Comment by thatbudakguy** [comment]
I totally though "Mon." was "Monday", indicating day of the week...my bad!

Is it worth including the "Published: " text, also? I get that this clarifies that it isn't necessarily the date it was written, etc, but if there's only one date I feel like it's reasonable to assume it's the publication date.

**Comment by gissoo** [comment]
@thatbudakguy I revised the grid of the header on this page based on the conversation we had yesterday regarding grids on the homepage and the single issue page for the issue number and the issue abstract ...so that on this page now the header is also centered. 
However what is different here is still the width. Because in general I'm not feeling great about the header being this close to the nav on the left as it is now on "Homepage" and "Single Issue" on mobile. I am even less comfortable with it on the article page, therefore the width is narrower. What if the header starts below the nav??

**Comment by gissoo** [comment]
@thatbudakguy I agree that published is not necessary, can drop it.

**Comment by thatbudakguy** [comment]
re: space next to the home/issue buttons, it seems like's there's plenty of space on mobile - those margins look fine to me. I was talking about relaxing the margins on tablet, since there's even more space to work with.

not sure what you mean about the header starting below the nav, but if i understand correctly, that would place the home/issue buttons lower down on the page (but the exact position would depend on how long the content at the top of the article was, which seems unpredictable). that might look odd after you start scrolling, and at the very least it would move the nav to a different position than it is on  other pages.

**Comment by rlskoeser** [comment]
Why are author names comma delimited instead of one per line? Seems like it would be more readable one per line, and more consistent with how they are displayed on the issue page.

Could you revise to show how an actual DOI will be displayed on the page? The one we've reserved for the Data Beyond Vision essay is **10.5281/zenodo.3713671** if you want to use that.

I wonder if the PDF and TXT links should look different or more prominent in some way? They look just like regular links.

I don't know if it's something we're looking at as part of this design issue or if it falls under the contextual note issue, but the footnote marker looks really small. Is it a big enough target per accessibility guidelines?

**Comment by gissoo** [comment]
@thatbudakguy Ok thanks for clarifying. The margins on the article page look fine to me right now (as far as how they are spaced) but definitely not fine to me on the homepage and the single issue page design-wise. 

I like the idea of relaxing the margins on tablet. However, if I do that on this page, it should only apply to this page and not the homepage and the single issue page. I strongly recommend that we don't apply this anywhere else. 

– No what I mean is that the nav will remain where it is, but the header (meaning all the titles and metadata) will begin below the nav, so there can be more relaxed margins on mobile as well (to save space)

**Comment by gwijthoff** [comment]
@thatbudakguy not sure what you mean by the "Published: " text?

@gissoo my reasoning behind not wanting to include the day of the month is just a matter of publishing convention. Newspapers (and sometimes magazines) include publication dates like April 11, 2020, but academic journals have only ever been published with dates like Spring 2020 or April 2020. To include a specific day would give the site more of a magazine/blog feel than the academic journal feel that we're going for.

P.S. I love that the arbitrary date we've chosen in this thread just happens to be my birthday :)

**Comment by gissoo** [comment]
@rlskoeser 

- Authors are separated by comma because on mobile it will make the page too long, it's already going to be a long page, I really don't want to make this page longer than it is when it doesn't have to be. I think it is readable as it is here. 

I can revise the DOI, will use your example, thanks for providing that!

I think we agreed that the links would look the same – I was working to make them more prominent and you said they can just be links. I do not think they should be larger or more prominent if we have agreed they can be links instead of buttons.

Regarding the footnote marks, let's not discuss that here, I'm not focusing on this aspect anymore right now as we agreed, I can erase this from here (I didn't because it creates some extra work for me later) so that we won't discuss it/look at it.

**Comment by gissoo** [comment]
@gwijthoff I see. I think I asked because I was looking at the PUL scholarly articles on PUL as well as how they are cited and saw the day of the month. I can erase the day then. 

(Note: what NB meant by "published" is I had this "Published Mon, 00, 0000" and I just deleted it, sorry, shouldn't revise designs till everyone has had a chance to look. – this is something that's happening because you're looking at the file I'm working on, rather than the Sketch --> Zeplin workflow we had before ...)

Woah!!! – I totally didn't even think about the date for a second!! :D Interesting ....(Some things have been happening with me in the past couple of days where I unknowingly do things that mean something more than I intend, starting to feel a bit scared now :D)

**Comment by rlskoeser** [comment]
My expectation is that we would have an exact day in the article citation / metadata but only display month/year on the website (I think at least some of the article metadata formats require a full date). @gwijthoff does that make sense to you?

@gissoo sorry, I forgot we'd previously discussed the txt/pdf link style. I agree about not having them be buttons.

@gissoo my concern about the author list was that it doesn't feel as fair to all authors, since some names seem more prominent/readable than others in that layout. But I defer to you.

**Comment by thatbudakguy** [comment]
@gissoo sorry I didn't understand about the nav, I got it backwards - thx for explaining. I think it's fine to relax the margins on tablet on this page only. however, in our discussion yesterday we talked about doing it on the homepage as well, and I showed you what that looked like. are we still OK to do that?

**Comment by gwijthoff** [comment]
@rlskoeser this is an interesting question. I always find it annoying when I pull a citation using Zotero and then have to manually delete the days from the bibliography it generates because academic citation styles don't typically include days, and the days that *are* generated seem to arbitrarily be the first or last of the month.

For example, I just generated these Chicago style citations from Zotero:

> Smith, Caleb. “Disciplines of Attention in a Secular Age.” Critical Inquiry 45, no. 4 (June 1, 2019): 884–909.
> Tenen, Dennis Yi. “The Emergence of American Formalism.” Modern Philology 117, no. 2 (November 1, 2019): 257–83.
> Parks, Lisa. “Televisual Epistemologies and Beyond.” Journal of Visual Culture 18, no. 2 (August 1, 2019): 234–49.

More often than not (in my Zotero library at least), the day of the month seems to be autogenerated as the 1st or 31st of that month. Which either means a) the journals didn't provide a day of the month and Zotero metadata requires one so automatically enters the 1st or 31st, or b) the journals themselves arbitrarily entered "1" or "31" because they didn't have an exact date.

So, maybe it is best to include a day for Zotero metadata purposes (just to be accurate) and only display Month-Year for users of the website, even if that means copyeditors and perfectionists like me might need to delete the days from the bibliographies that Zotero exports.

tl;dr: The ideal solution, from my perspective, would be to have no have no day in the metadata. Unless that means Zotero will automatically enter "1" or "31" of no day is provided, in which case day for metadata, month-year for site users.

**Comment by rlskoeser** [comment]
@gwijthoff that's fascinating! I don't think Zotero is very particular or opinionated about dates (although I could be wrong), but I _think_ the metadata format that I was planning to use (that both Zotero and Google Scholar can consume) require full dates. Let's keep this in mind when we work on and test #29 .

**Comment by gissoo** [comment]
@rlskoeser Thanks for writing. 
- I did make the pdf, txt links more prominent by making the DOI link lighter/consistent with the author styles. 
- Let me think/try it for the author names

**Comment by gissoo** [comment]
@thatbudakguy Let's discuss again. I think I already revised that page based on our conversation yesterday, but let's double check with your version.

**Comment by gissoo** [comment]
@gwijthoff @rlskoeser @thatbudakguy Please let's reach an agreement here as well on how we want to display the dates, so I can submit the design for this issue as well. Thank you!!

**Comment by rlskoeser** [comment]
@gissoo I thought it was decided, but I see how the conversation how that might be unclear! My understanding is that the display version should only be month/year per @gwijthoff comments. The citation metadata (which is not visible on the page anywhere) may require an exact day and not just month/year, but it doesn't impact the date we display on the page.

**Comment by gissoo** [comment]
@rlskoeser thank you for writing! 
@gwijthoff I have updated the designs on this page to reflect this change in the dates. Please take a look and if you're ok please close this issue. (Note: The dates are only updated on the designs attached to this github issue, and may not be updated on the other pages yet – there are many screens on the other issues that are still open, updating small changes on all those screens is not an efficient approach – dates will be updated in issues 43 and 57 once we agree on the designs in those issues)

**Comment by gwijthoff** [comment]
@gissoo the date is perfect now, thanks! This is the right Month/Year format, and we'll include day in the underlying metadata as necessary. And, as you say above with date not changed yet in issues 43 and 57, I assume here that "DOI:" will be included before the hyperlink as it is currently in 43 and 57, right?

**Comment by gissoo** [comment]
@gwijthoff we have not reached an agreement on the DOI yet, per your comment on the other issue. I do think it is fine without the label, also referring to your comment on issue 57. @rlskoeser @gwijthoff @thatbudakguy is that ok with you? (i.e. it will match what you see on the designs attached here). Please let me know if there are strong objections, I'm flexible about this, just have slight preferences for the reasons/questions I brought up about it in my comment on issue 57. (my 4th comment from the bottom of 57)

**Comment by gwijthoff** [comment]
@gissoo sorry, I'm afraid I'm confusing things here—there's a lot of correspondence to catch up on! I thought the debate was whether the abbreviation "DOI" should also include some kind of explainer on what that is, that it means Digital Object Identifier, etc. I leaned no on that and don't think it's necessary. But I *do* think that "DOI:" needs to be included, otherwise most people won't know what that link is. Right now it's just a string of numbers and the word "zenodo."

**Comment by rlskoeser** [comment]
I agree, DOI needs to be included. I thought it was just a convention, but I went looking and actually it's more than that. The correct forms are doi:10.### or a full url.  

See https://www.doi.org/doi_handbook/2_Numbering.html#2.6.1 and 
https://blog.apastyle.org/apastyle/2014/07/how-to-use-the-new-doi-format-in-apa-style.html

**Comment by gissoo** [comment]
@gwijthoff Thanks for writing, I was making a decision based on your other comment, this is clear now. Sure, that's fine to include "doi". 

@rlskoeser thank you for the links, will look into it. Assuming "doi" won't be a link. If this is the case, then we will need one more text style @thatbudakguy just so you know.

**Comment by rlskoeser** [comment]
I would make the full label doi:10.### a link - the `doi:` is part of the identifier. (I was just reading up and it isn't actually a URN because DOI isn't registered as a namespace ... but I guess that isn't really relevant here...)

**Comment by gissoo** [comment]
@rlskoeser @gwijthoff @thatbudakguy I read through the links RSK provided and made the change with the DOI. 
Does the DOI link look ok now?

**Comment by rlskoeser** [comment]
Looks great to me.

**Comment by gwijthoff** [comment]
Perfect! I think we're good to go on this.
